pkgs/top-level/python-packages.nix

```diff
@@ -16241,6 +16272,24 @@ in {
       };
     in if isPy3k then py3 else py2;
 
+  pyqrcode = buildPythonPackage rec {
+    name = "${pname}-${version}";
+    pname = "PyQRCode";
+    version = "1.2.1";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "1m9ln8k9v7dfbh1i81225hx5mdsh8mpf9g7r4wpbfmiyfcs7dgzx";
+    };
+
+    meta = {
+      description = "A QR code generator written purely in Python with SVG, EPS, PNG and terminal output";
+      home = "https://pypi.python.org/pypi/PyQRCode/";
```

Comment:
`homepage`, no quotes. And the homepage should be https://github.com/mnooner256/pyqrcode.